src/Apitizer/Types/Filter.php

```diff
@@ -216,6 +206,33 @@ public function setValue($value): self
         return $this;
     }
 
+    public function setType(string $type): self
+    {
+        $this->type = $type;
+        return $this;
+    }
+
+    public function setExpectArray(bool $expectArray): self
+    {
+        $this->expectArray = $expectArray;
+        return $this;
+    }
+
+    public function setFormatting(string $format): self
+    {
+        $this->format = $format;
+        return $this;
+    }
+
+    /**
+     * @param array<string> $enums, This is used to check whether the value is an available option.
+     */
+    public function setEnumerators(array $enums): self
+    {
+        $this->enums = $enums;
+        return $this;
+    }
```

Comment:
Could you add an `@interal` to the docblocks? For example:

```
/**
 * @internal
 */
```

This signals users of the filter that these are not meant to be publicly used, even though they are public methods.

Reply:
Added on all setters